README.md

```diff
@@ -23,180 +23,5 @@ Kaito has the following key differentiations compared to most of the mainstream
-For example, to enable 4-bit quantization on a `falcon-7b-instruct` deployment, you would execute:
-
-```sh
-kubectl edit deployment workspace-falcon-7b-instruct
-```
-
-Within the deployment specification, locate and modify the command field.
-
-#### Original
-
-```sh
-accelerate launch --num_processes 1 --num_machines 1 --machine_rank 0 --gpu_ids all inference_api.py --pipeline text-generation --torch_dtype bfloat16
-```
-
-#### Modify to enable 4-bit Quantization
-
-```sh
-accelerate launch --num_processes 1 --num_machines 1 --machine_rank 0 --gpu_ids all inference_api.py --pipeline text-generation --torch_dtype bfloat16 --load_in_4bit
-```
-
-Currently, we allow users to change the following paramenters manually:
-
-- `pipeline`: For text-generation models this can be either `text-generation` or `conversational`.
-- `load_in_4bit` or `load_in_8bit`: Model quantization resolution.
-
-Should you need to customize other parameters, kindly file an issue for potential future inclusion.
-
-### What is the difference between instruct and non-instruct models?
-
-The main distinction lies in their intended use cases. Instruct models are fine-tuned versions optimized
-for interactive chat applications. They are typically the preferred choice for most implementations due to their enhanced performance in
-conversational contexts.
-On the other hand, non-instruct, or raw models, are designed for further fine-tuning.
-
-## Contributing
-
-[Read more](docs/contributing/readme.md)
-<!-- markdown-link-check-disable -->
-This project welcomes contributions and suggestions. The contributions require you to agree to a
-Contributor License Agreement (CLA) declaring that you have the right to, and actually do, grant us
-the rights to use your contribution. For details, visit [CLAs for CNCF](https://github.com/cncf/cla?tab=readme-ov-file).
-
-When you submit a pull request, a CLA bot will automatically determine whether you need to provide
-a CLA and decorate the PR appropriately (e.g., status check, comment). Simply follow the instructions
-provided by the bot. You will only need to do this once across all repos using our CLA.
-
-This project has adopted the CLAs for CNCF, please electronically sign the CLA via
-https://easycla.lfx.linuxfoundation.org. If you encounter issues, you can submit a ticket with the
-Linux Foundation ID group through the [Linux Foundation Support website](https://jira.linuxfoundation.org/plugins/servlet/desk/portal/4/create/143).
-
-## License
-
-See [MIT License](LICENSE).
-
-## Code of Conduct
-
-KAITO has adopted the [Cloud Native Compute Foundation Code of Conduct](https://github.com/cncf/foundation/blob/main/code-of-conduct.md). For more information see the [KAITO Code of Conduct](CODE_OF_CONDUCT.md).
-
-<!-- markdown-link-check-enable -->
-## Contact
-
-"Kaito devs" <kaito-dev@microsoft.com>
-
```

Comment:
Do we need to keep these sections in the README ? I remember this was added after joining to cncf. 
@sdesai345  @Fei-Guo

Reply:
I'll revert to keep these for now. I rather merge this asap to avoid merge conflicts, then we can make further adjustments.